src/client/components/pages/parts/revisions-table.js

```diff
@@ -125,11 +131,13 @@ function RevisionsTable(props) {
 
 RevisionsTable.propTypes = {
 	results: PropTypes.array.isRequired,
+	showEntities: PropTypes.bool,
 	showRevisionEditor: PropTypes.bool,
 	showRevisionNote: PropTypes.bool,
-	tableHeading: PropTypes.string
+	tableHeading: PropTypes.any
```

Comment:
According to https://reactjs.org/docs/typechecking-with-proptypes.html you can use `PropTypes.node`:
```
Anything that can be rendered: numbers, strings, elements
or an array (or fragment) containing these types.
```

It's more appropriate to describe what we're expecting, like a string or a JSX/React element like the <div> used for the header.

Reply:
I'll change it